src/docs/cookbook-payments/customize-pricing/index.md

```diff
@@ -466,409 +270,198 @@ The input can be styled by adding the following style definitions to
 }
 ```
 
-The next task is to modify the `EstimatedBreakdownMaybe` component to
-support the cleaning fee and add a new line item type to the booking
-breakdown.
-
-```shell
-└── src
-    └── forms
-        └── BookingDatesForm
-            └── EstimatedBreakdownMaybe.js
-```
-
-In the `EstimatedBreakdownMaybe` component:
-
-- Import `LINE_ITEM_CLEANING_FEE` from `util/types.js`
-- Spread the `cleaningFee` out of `props.bookingData`:
-
-  ```js
-    const EstimatedBreakdownMaybe = props => {
-      const { unitType, unitPrice, startDate, endDate, quantity, cleaningFee } = props.bookingData;
-  ```
-
-- Add `cleaningFee` to the parameter list in the `estimatedTransaction`
-  function definition and invocation.
-
-  <!-- prettier-ignore -->
-  ```js
-  const tx = estimatedTransaction(unitType, startDate, endDate, unitPrice, quantity, cleaningFee);
-  ```
+### Capture add-ons value for the booking breakdown
 
-Now the `estimatedTotalPrice` function can be renamed as
-_"estimatedRentalPrice"_:
+In the `BookingDatesForm` `handleOnChange` function, resolve if the
+cleaning fee is selected and pass the information to the
+`onFetchTransactionLineItems` function:
 
 ```diff
-- const estimatedTotalPrice = (unitPrice, unitCount) => {
-+ const estimatedRentalPrice = (unitPrice, unitCount) => {
-    const numericPrice = convertMoneyToNumber(unitPrice);
-    const numericTotalPrice = new Decimal(numericPrice).times(unitCount).toNumber();
-    return new Money(
-      convertUnitToSubUnit(numericTotalPrice, unitDivisor(unitPrice.currency)),
-      unitPrice.currency
-    );
-  };
-```
-
-Then create a new `estimatedTotalPrice` function that includes
-`cleaningFee`:
+ handleOnChange(formValues) {
+   const { startDate, endDate } =
+     formValues.values && formValues.values.bookingDates ? formValues.values.bookingDates : {};
+   const listingId = this.props.listingId;
+   const isOwnListing = this.props.isOwnListing;
+
++  const addOns =
++    formValues.values && formValues.values.addOns ? { addOns: formValues.values.addOns } : {};
+
+   if (startDate && endDate && !this.props.fetchLineItemsInProgress) {
+     this.props.onFetchTransactionLineItems({
+-      bookingData: { startDate, endDate },
++      bookingData: { startDate, endDate, ...addOns },
+       listingId,
+       isOwnListing,
+     });
+   }
+ }
+```
+
+There, now the listing page booking panel indicates if a listing has a
+cleaning fee configured and it can be selected when making a booking.
+Next, let's move to the server side and update our pricing parameters.
```

Comment:
```suggestion
Next, let's move to the server-side and update our pricing parameters.
```

There are other places as well where server-side and client-side are used either with or without a hyphen. We should unify how we use these, and I think, the hyphen should be the way to go.